src/Framework/FieldsAPI/PaymentGateways.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Give\Framework\FieldsAPI;
+
+class PaymentGateways extends Field
+{
+    public function getNodeType(): string
+    {
+        return 'layout';
+    }
+
+    public function getType(): string
+    {
+        return 'gateways';
```

Comment:
This method is inherited from the base node type class, which expects a class constant of `TYPE` where the type should be defined in each class.

Reply:
Ah! Ok. Resolved in eb3fb7b